tests/cxx/badinc.cc

```diff
@@ -1841,7 +1841,6 @@ int main() {
   // IWYU: I1_Class needs a declaration
   H_TemplateFunction<I1_Class*>(&i1_class);
   H_TemplateFunction(&i1_class);
-  // IWYU: I1_Enum is...*badinc-i1.h
```

Comment:
And it's nice we don't need `allocator` for `string` because we don't use allocator explicitly.